Sources/OpenAPIRuntime/Conversion/Converter+Server.swift

```diff
@@ -134,7 +134,7 @@ extension Converter {
                         dateTranscoder: configuration.dateTranscoder
                     )
                 )
-                let value = try decoder.decode(
+                let value = try decoder.decodeIfPresent(
```

Comment:
This is the core of the fix - notice we're in `getOptionalQueryItemAsURI`, but we were decoding unconditionally here.